secure_message/resources/threads.py

```diff
@@ -109,7 +109,8 @@ def get():
 
         logger.info("Successfully retrieved threads for user", user_uuid=g.user.user_uuid)
         messages, links = process_paginated_list(result, request.host_url, g.user, message_args, THREAD_LIST_ENDPOINT)
-        messages = add_users_and_business_details(messages)
+        if messages:
```

Comment:
@insacuri  Could be a good idea but I could also see it potentially polluting the splunk logs if the user keeps looking at empty lists.

Reply:
Don't worry about polluting the splunk logs, better to have too much logging and strip it back then not have enough when a problem occurs.  A middle ground option could be to log out a line saying 'No messages retrieved' or something to that effect and not bother logging out the args. I think we can get some useful url param information from another log line that gets logged out every time the service gets hit.  

Feel free to not bother with it though because it's not like we're logging 'Retrieved X messages/threads' for the non-zero cases, so that asymmetry might be weird.